### DISCUSSION TITLE
Clearly date all relevant documentation, tutorials, code examples, etc.

### DISCUSSION THREAD

**Discussion by vladchuk** [FilledStacks/stacked#925] [General]
A very long time ago, when FilledStacked was only conceived, I asked Dane to timestamp videos and tutorials, and he graciously agreed. I'd like to request the same for all existing and new content, so that we always know:

- how new/old the info is - helps with relevance
- when/if something was modified - helps to identify corrective steps and actions, if any
- how busy Dane has been :) - so that we can encourage him to keep it up!

Thanks!

**Comment by FilledStacks** [comment]
This is quite a large undertaking and most likely will not benefit more than a few users. If Stacked wasn't completely paid for by me, built and maintained by me, with 0 support in terms of finance or community led code, for a tool that I know for a fact companies are making thousands of dollars with a month, then I'd probably be more open to taking this on.

I won't be adding this additional step to my free creation process. Maintaining 11 packages, the docs for it, answering questions about it, writing tutorials for it, making youtube videos for it while trying to restart my business and stop burning my savings on providing free and extremely valuable products and content, is _**difficult**_ to put it in the lightest terms I possibly can. 

It's a great request and will be very beneficial for the community and most likely the trust in the product. Hopefully everyone can come together and make it happen 🚀

Like everything else in stacked the [docs are open source](https://github.com/FilledStacks/stacked-docs) so I'd be happy if you add in there what you're talking about.

**Comment by vladchuk** [comment]
Understood. If it's not too difficult to add to the new content, perhaps you can keep it in mind?..
